domains/osenoye.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "erickvenom99",
+    "email": "osenoye.aghaulor@gmail.com"
+  },
+  "records": {
+    "TXT": "google-site-verification=XQ2SWp81z61JnPZlPu-kAe4BEHwT5PNep5OKK0A-uGQ"
+  },
+  "proxied": false
```

and since you cant mix up CNAME unless proxied
```suggestion
  "proxied": true
```